Test/logger/CSVLogger.dfy

```diff
@@ -6,7 +6,7 @@
 // wasn't populating the resource count test result property, and the CSV logger was
 // defaulting to 0 for that column.
 // CHECK-NOT: Impl\$\$_module.__default\.ExampleWithSplits\$\$.*,Passed,.*,0
-// CHECK-NEXT: Impl\$\$_module.__default\.ExampleWithSplits\$\$1,Passed,.*,.*
+// CHECK: Impl\$\$_module.__default\.ExampleWithSplits\$\$1,Passed,.*,.*
```

Comment:
Can you please explain this?

Reply:
The order of elements in the XML has become non-deterministic due to the concurrency, and CHECK-NEXT is very specific about the order.